lib/node_modules/@stdlib/ndarray/README.md

```diff
@@ -70,21 +70,6 @@ In addition, the namespace contaians the following multidimensional array utilit
 
 <div class="namespace-toc">
 
--   <span class="signature">[`ndarrayCastingModes()`][@stdlib/ndarray/casting-modes]</span><span class="delimiter">: </span><span class="description">list of ndarray casting modes.</span>
```

Comment:
@Planeshifter I am not certain why these lines are being deleted. Based on the Git history for `README.md` and `lib/index.js`, I don't see an immediate reason why the README contents should change. Perhaps you know?

Reply:
@Planeshifter I'm guessing that, until we determine what's going on here, we won't be able to merge namespace ToC PRs, as we'll keep seeing this undesired change.

Reply:
I think the issue is [dispatch-by](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/ndarray/dispatch-by) missing a `README.md` file; my hunch is that we have something fail in the process due to that.

Reply:
Also noticed that in a line above, we have a spelling mistake: `contaians`.

Reply:
Fixed!

Reply:
After adding a skeleton `README.md` for `dispatchBy`, the workflow didn't erase any of the `ndarray` table of contents anymore! We could try to dig into this and guard against this case, but should be easy to avoid going forward, too.

Reply:
@Planeshifter we should add a todo to the todo tracker. We cannot guarantee that when a package is a WIP that we'll have all files, including a README, nor should we have to. E.g., what if we're actively working on a package and a namespace ToC gets opened? We shouldn't be forced to add a README first because of undocumented institutional knowledge. It is fine for us because we can store all these things in our head, but not for others who may not have that knowledge. In general, our workflows should be robust against failure modes, and I don't think that this is an exception.